package.json

```
@@ -67,8 +67,8 @@
   },
   "peerDependencies": {
     "prop-types": "^15.7.2",
-    "react": ">=16.13.0",
-    "react-dom": ">=16.13.0"
+    "react": ">=18.2.0",
+    "react-dom": ">=18.2.0"
```

```suggestion
    "react": ">=16.13.0",
    "react-dom": ">=16.13.0"
```

This ensures projects using >= v16 of React can still install newer versions of cdbreact without getting a peer dependency error.